gulpfile.js

```diff
@@ -21,51 +21,49 @@ gulp.task('sass', function() {
 });
 
 gulp.task('watch', function () {
-  gulp.watch('src/**/*.scss', ['sass']);
+  gulp.watch('src/**/*.scss', gulp.series('sass'));
 });
 
 gulp.task('hugo-server', function (cb) {
-  let hugo_process = exec(
-    hugo, ['server', '--source=docs', '--disableFastRender'],
-    function (err, stdout, stderr) {
-      console.log(stderr);
-      return err ? cb(err) : cb();
-    }
-  );
+  const flags = ['server', '--source=docs', '--disableFastRender'];
+  let hugo_process = exec(hugo, flags, function (err, stdout, stderr) {
+    console.log(stderr);
+    return err ? cb(err) : cb();
+  });
+
   hugo_process.stdout.pipe(process.stdout);
   return hugo_process;
 })
 
 gulp.task('hugo-build', function (cb) {
-  let hugo_process = exec(
-    hugo, ['--source=docs'],
-    function (err, stdout, stderr) {
-      console.log(stderr);
-      return err ? cb(err) : cb();
-    }
-  );
+  const flags = ['--source=docs'];
+  let hugo_process = exec(hugo, flags, function (err, stdout, stderr) {
+    console.log(stderr);
+    return err ? cb(err) : cb();
+  });
+
   hugo_process.stdout.pipe(process.stdout);
   return hugo_process;
 })
 
 gulp.task('minify-css', () => {
   return gulp.src('src/**/*.scss')
-  .pipe(sass({outputStyle: 'compressed'}).on('error', sass.logError))
-  .pipe(autoprefixer())
-  .pipe(rename('paper.min.css'))
-  .pipe(gulp.dest('dist'))
-  .pipe(gulp.dest('docs/static/assets'));
+    .pipe(sass({outputStyle: 'compressed'}).on('error', sass.logError))
+    .pipe(autoprefixer())
+    .pipe(rename('paper.min.css'))
+    .pipe(gulp.dest('dist'))
+    .pipe(gulp.dest('docs/static/assets'));
 });
 
 gulp.task('lint-css', function() {
   return gulp.src('src/**/*.scss')
     .pipe(gulpStylelint({
       reporters: [
-        {formatter: 'string', console: true}
-      ]
+        { formatter: 'string', console: true },
+      ],
     }));
 })
 
-gulp.task('default', ['sass','watch','hugo-server']);
-gulp.task('build', ['sass','minify-css','hugo-build']);
-gulp.task('postinstall', ['sass','minify-css']);
+gulp.task('default', gulp.series('sass', 'minify-css', 'hugo-build'));
```

Comment:
Ah nice, so `gulp.series` is the replacement for passing in an array?

Reply:
Absolutelty, there are `gulp.series` and `gulp.parallel`. I've followed [this simple guide](https://fettblog.eu/gulp-4-parallel-and-series/) for the migration.

Reply:
Awesome!